### PR TITLE
Fix minor issues

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: src/runtime/EnumerateObject.cpp [hunk 1]
```diff
@@ -201,6 +201,7 @@ void EnumerateObjectWithIteration::executeEnumeration(ExecutionState& state, Enc
     m_hiddenClassChain.push_back(m_object->structure());
 
     std::unordered_set<String*, std::hash<String*>, std::equal_to<String*>, GCUtil::gc_malloc_allocator<String*>> keyStringSet;
+    std::unordered_set<Object*> protoChainSet;
```
Comment (clover2123) [inline src/runtime/EnumerateObject.cpp:204]: It seems that this `protoChainSet` is defined but not used at all

File: src/runtime/EnumerateObject.cpp [hunk 2]
```diff
@@ -222,8 +223,14 @@ void EnumerateObjectWithIteration::executeEnumeration(ExecutionState& state, Enc
         }
         m_hiddenClassChain.push_back(proto->structure());
         proto = proto->getPrototypeObject(state);
+
+        // v8 throw exception when there is too many things on prototype chain
+        if (UNLIKELY(m_hiddenClassChain.size() > 1024 * 128)) {
```
Comment (clover2123) [inline src/runtime/EnumerateObject.cpp:228]: What is this number `1024*128`?
v8 benchmark arbitrarily sets this threshold number to check huge prototype chains?

File: src/runtime/EnumerateObject.cpp (hunk 1 above)
Comment (ksh8281) [reply src/runtime/EnumerateObject.cpp:204]: Oops

File: src/runtime/EnumerateObject.cpp (hunk 2 above)
Comment (ksh8281) [reply src/runtime/EnumerateObject.cpp:228]: enabled v8 test has infinity prototype chain with ProxyObject.
ordinary object prevents infinity prototype when setting __proto__,
but there is no limitation for ProxyObject.
1024 * 128 is practical limit

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**